jupyter_client/session.py

```diff
@@ -807,6 +807,10 @@ def send(
             # ZMQStreams and dummy sockets do not support tracking.
             track = False
 
+        if isinstance(stream, zmq.asyncio.Socket):
+            assert stream is not None
```

Comment:
I assume leftover debug, this will never be None if it's an asyncio.Socket

```suggestion
```

Reply:
This is there to satisfy `mypy`

Reply:
type checkers, always making code better!